docs/data/data-grid/components/quick-filter/quick-filter.md

```diff
@@ -42,20 +48,47 @@ It renders the `baseTextField` slot.
 `<QuickFilterClear />` is a button that resets the filter value.
 It renders the `baseIconButton` slot.
 
+### Quick Filter Trigger
+
+`<QuickFilterTrigger />` is a button that expands/collapses the quick filter.
+It renders the `baseIconButton` slot.
+
 ## Custom elements
 
 Use the `render` prop to replace default elements.
 See [Components usage—Customization](/x/react-data-grid/components/usage/#customization) for more details, and [Toolbar—Custom elements demo](/x/react-data-grid/components/toolbar/#custom-elements) for an example of a custom Quick Filter.
 
+## Recipes
+
+Below are some ways the Quick Filter component can be used.
+
+### Default expanded state
+
+The quick filter is uncontrolled by default and can be toggled via the trigger. The `defaultExpanded` prop can be used to set the default expanded state.
+
+{{"demo": "GridUncontrolledQuickFilter.js", "bg": "inline", "defaultCodeOpen": false}}
+
+### Expand quick filter via keyboard
+
+The demo below shows how to control the quick filter state using the `expanded` and `onExpandedChange` props to support expanding the quick filter via keyboard. Try pressing <kbd class="key">Cmd</kbd> (or <kbd class="key">Ctrl</kbd> on Windows)+<kbd class="key">Shift</kbd>+<kbd class="key">F</kbd>.
```

Comment:
TODO: Change this keyboard shortcut, it full-screens the browser window in Firefox.